plugins/net_plugin/test/main.cpp

```diff
@@ -0,0 +1,41 @@
+#include <cstdlib>
+#include <iostream>
+#include <boost/test/included/unit_test.hpp>
+#include <fc/log/logger.hpp>
+#include <eosio/chain/exceptions.hpp>
+
+//extern uint32_t EOS_TESTING_GENESIS_TIMESTAMP;
+
+void translate_fc_exception(const fc::exception &e) {
+   std::cerr << "\033[33m" <<  e.to_detail_string() << "\033[0m" << std::endl;
+   BOOST_TEST_FAIL("Caught Unexpected Exception");
+}
+
+boost::unit_test::test_suite* init_unit_test_suite(int argc, char* argv[]) {
+   // Turn off blockchain logging if no --verbose parameter is not added
```

Comment:
this sentence sounds awkward. Maybe something like "Turn off logging if --verbose parameter was not provided" ?

Reply:
Reworded the sentence